client/templates/response/_partials/update-header.njk

```diff
@@ -52,10 +52,12 @@
                     Request juror info by post
                   </a>
 
-                  {% set sendToUrl = url('response.assign.get', { id: response.jurorNumber }) + "?reply_method=" + method %}
-                  <a id="processSendTo" href="{{ sendToUrl }}" role="button" draggable="false" class="govuk-button moj-button-menu__item govuk-button--secondary " value="send_to" data-module="govuk-button">
-                    Send to a colleague...
-                  </a>
+                  {% if response.assignedStaffMember.login === authentication.login or authentication.staff.rank === 1 %}
```

Comment:
we should not use rank anymore... we should use the `isManager` or `isTeamLead` rules.

Reply:
@ricardofreitasrocha thank you. That's fixed now.